internal/tracker/tracker.go

```diff
@@ -33,6 +34,29 @@ import (
 
 const ociPrefix = "oci://"
 
+const jsonSchema = `{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "patternProperties": {
+        ".*": {
+            "type": "array",
+            "items": {
+                "type": "object",
+                "properties": {
+                    "effective_on": {"type": "string"},
+                    "expires_on": {"type": "string"},
+                    "ref": {"type": "string"}
+                },
+                "required": ["effective_on", "ref"],
+                "additionalProperties": false
+            },
+            "uniqueItems": true,
```

Comment:
Not sure what the probability of encountering duplicate value is, if it is near certain that it will happen then we'll hit this and prevent new Tasks from being pushed to quay.